device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,36 @@
+{%- set default_cable = '5m' %}
+
+{%- include 'buffer_ports.j2' %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "121964196",
+            "type": "ingress",
+            "mode": "dynamic",
+            "xoff": "29520896"
+        },
+        "egress_lossless_pool": {
+            "size": "121964196",
+            "type": "egress",
+            "mode": "static"
+        }
+    },
+    "BUFFER_PROFILE": {
+        "ingress_lossy_profile": {
+            "pool": "ingress_lossless_pool",
+            "size": "0",
+            "static_th": "167546528"
+        },
+        "egress_lossy_profile": {
+            "pool": "egress_lossless_pool",
+            "size": "1778",
+            "dynamic_th": "1"
+        },
+        "egress_lossless_profile": {
+            "pool": "egress_lossless_pool",
+            "size": "0",
+            "static_th": "125995684"
```

Comment:
why is this larger than the egress_lossless pool size?

Reply:
I checked the configuration and also the update today. This setting is indeed larger than egress pool. I have replied the CSP case to confirm the reason, hopefully will get answers too.